files/manage_bootstrap_user.py

```diff
@@ -18,7 +18,11 @@
 eggs.require("mercurial")
 
 import galaxy.config
-from galaxy.web import security
+
+try:
+    from galaxy.web.security import SecurityHelper as Security
```

Comment:
Can you switch this around so that the new import is tried first, and add a comment in the except clause that this maintains backwards compatibility with galayx verisons < 19.05 ? fwiw the PR that modified this was https://github.com/galaxyproject/galaxy/pull/7560

Reply:
Ok, this is done, together with a bit of code cleaning